server/api/hook.go

```diff
@@ -139,6 +139,7 @@ func PostHook(c *gin.Context) {
 
 	if build.Event == model.EventPull && !repo.AllowPull {
 		logrus.Infof("ignoring hook. repo %s is disabled for pull requests.", repo.FullName)
+		c.Writer.Write([]byte("pulls are disabled on woodpecker for this repo"))
```

Comment:
So this text can for example be seen in Github under delivered webhooks, right?

Reply:
yes - it wont show up in status - but if you are admin and look at the web-hook delivery panel you will see this message

Reply:
@anbraten example:
![Bildschirmfoto zu 2021-09-27 23-30-23](https://user-images.githubusercontent.com/24977596/134988173-71fcbfc7-eb67-4fb6-9bd8-a6651ae9b398.png)